README.md

```diff
@@ -195,7 +195,7 @@ ActiveJob.jobs.pending.where(job_class: "SomeJob").limit(10).offset(5)
 
 # For adapters that support these statuses:
 # All scheduled/in-progress/finished jobs of a given class
-ActiveJob.jobs.scheudled.where(job_class: "SomeJob")
+ActiveJob.jobs.scheduled.where(job_class: "SomeJob")
```

Comment:
Ooh, thank you! Also, this should be `job_class_name` instead of `job_class`! We had these examples from before we changed the parameter name to be consistent across the board, but I forgot to change the examples themselves. Would you mind changing also these? 🙏

Reply:
@rosa updated

Reply:
So nice, thank you so much 🙏